docs/INSTALL.md

```diff
@@ -18,6 +18,7 @@ proceed.
 A normal Unix installation is made in three or four steps (after you've
 unpacked the source archive):
 
+    ./buildconf
```

Comment:
The documentation presumes a standard build is done by using a downloaded curl release tarball. They come with a configure script already made and removes the need for all the autotools etc being present locally that buildconf otherwise pushes to the person wanting to build curl.

If you look at just the paragraph above the one you want to change, it mentions specifically what to do if you're building code directly off git. If anything should be added for this, I would think it belongs under the "from git" header.

Won't you agree?

Reply:
Ah, I'm stupid. It's even in `GIT-INFO`. Thanks!